src/sass/_config.scss

```diff
@@ -80,6 +80,13 @@ $fontSizes: (
   header: 3rem
 );
 
+$bitSizes: (
```

Comment:
I would keep it in bit scss ;)

Reply:
@aju why? I think would be better to keep config for all types of texts in one place?